src/components/studies/participants/BatchEditForm.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import { Dialog } from '@material-ui/core'
+import { EditParticipantForm } from './ParticipantForms'
+import { EditDialogTitle } from './ParticipantTableGrid'
+import ParticipantService from '../../../services/participants.service'
+
+type BatchEditFormProps = {
+  isEnrolledById: boolean
+  isBatchEditOpen: boolean
+  setIsBatchEditOpen: Function
```

Comment:
since those are callback functions, it's customary to have them start with on e.g. onSetIsBatchEditOpen